short-domain-name-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,10 @@
+server:
+  port: 8080
+
+logging:
+  level:
+    root: info
+
+system:
+  maxKey: 10000
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 9b93eccd-6153-4c9b-abbb-b0acebcaa4de -->